geomesa-tools/bin/download-data.sh

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env bash
+
+type=$1
+
+setGeoHome ()
+{
+    SOURCE="${BASH_SOURCE[0]}"
+    # resolve $SOURCE until the file is no longer a symlink
+    while [ -h "${SOURCE}" ]; do
+        bin="$( cd -P "$( dirname "${SOURCE}" )" && pwd )"
+        SOURCE="$(readlink "${SOURCE}")"
+        # if $SOURCE was a relative symlink, we need to resolve it relative to the path where
+        # the symlink file was located
+        [[ "${SOURCE}" != /* ]] && SOURCE="${bin}/${SOURCE}"
+    done
+    bin="$( cd -P "$( dirname "${SOURCE}" )" && cd ../ && pwd )"
+    export GEOMESA_HOME="$bin"
+    export PATH=${GEOMESA_HOME}/bin:$PATH
+    echo >&2 "Warning: GEOMESA_HOME is not set, using $GEOMESA_HOME"
+}
+
+if [ -z "$GEOMESA_HOME" ]; then
+  setGeoHome
+fi
+
+if [ "$type" = 'gdelt' ]; then
```

Comment:
[picky; style] This may be a good opportunity to use a Bash-case statement in lieu of these nested if/then/else's.  They're equivalent, so there's no real need to change things.